docs/markdown/Cross-compilation.md

```diff
@@ -131,6 +132,12 @@ cross compilation you can set them with `[langname]_args =
 [args]`. Just remember to specify the args as an array and not as a
 single string (i.e. not as `'-DCROSS=1 -DSOMETHING=3'`).
 
+*Since 0.52.0* The `sys_root` property may point to the root of the host
+system path (the system that will run the compiled binaries). This is used
+internally by meson to set the PKG_CONFIG_SYSROOT_DIR environment variable
+for pkg-config. If this is unset the host system is assumed to share a root
+with the build system.
```

Comment:
Oh BTW this build vs host language can be modified, since there is no host machine-specific or cross=true-specific behavior in implementation (other than their being no `[properties]` section in the native file). I would use like lanaguage "default"/"ambient" sysroot vs manually-specified sysroot.